nano/lib/work.cpp

```diff
@@ -99,6 +101,12 @@ void nano::work_pool::loop (uint64_t thread)
 					blake2b_final (&hash, reinterpret_cast<uint8_t *> (&output), sizeof (output));
 					blake2b_init (&hash, sizeof (output));
 					iteration -= 1;
+
```

Comment:
I think this should be moved up one loop so it's not doing a branch check in the innermost loop.

Reply:
thanks, have moved it to the outside loop.